src/util/filteringStream.ts

```diff
@@ -0,0 +1,33 @@
+import stream from 'stream';
+
+/**
+ * Creates a writable stream that prevents HTTP logs from being logged
+ *
+ * It is done to filter out HTTP logs which are coming from Probot.
+ * Unfortunately there is no other way to disable these logs.
+ *
+ * @name filteringStream
+ * @function
+ * @param out {Stream} original stream
+ * @return {WritableStream} that you can pipe bunyan output into
+ */
+const filteringStream = (out) => {
+  const writable = new stream.Writable({
+    write: function(chunk, encoding, next) {
+      if (!shouldBeFiltered(chunk)) {
+        out.write(chunk, encoding)
+      }
+      next();
+    }
+  });
+  return writable
+}
+
+
+const shouldBeFiltered = (chunk: any) : boolean => {
```

Comment:
@mboudreau How can I fix this warning? What should I use if I don't know the "chunk" type?

Reply:
Isn't the chunk a string here?

Reply:
Pretty sure the `chunk` in this case is based on the `out` type, which I believe, since you're inputting the output of bunyan format, is going to be a string.

Reply:
That's what I thought in the beginning. But turned out that chunk has `Buffer` type here. I am using `any` because chunk has `any` type in `write` method parameter in `WritableOptions`